src/components/ActivitiesList.js

```diff
@@ -1,18 +1,23 @@
 import React from 'react';
-import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 import ActivitiesListItem from './ActivitiesListItem';
 import selectActivities from '../store/activities/selectors/activities';
 
 const ActivitiesList = ({ activities }) => (
   <div>
     <h1>Activities List</h1>
-    {activities.map((activity) => (
-      <ActivitiesListItem {...activity} key={activity.id}/>
+    {activities.map(activity => (
+      <ActivitiesListItem {...activity} key={activity.id} />
     ))}
   </div>
 );
 
-const mapStateToProps = (state) => ({
+ActivitiesList.propTypes = {
+  activities: PropTypes.array
```

Comment:
if prop is not required, we should add default prop. 
```
ActivitiesList.defaultProps = {
  activities: [],
}
```
it will prevent broken app (and that would happen if activities would be undefined.

Reply:
or change this prop to required (in this case not recommended, you will never know if some activities has been set up in the past and able to read)